pkgs/tools/networking/goflow/default.nix

```
@@ -0,0 +1,27 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "goflow";
+  version = "3.4.2";
+
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Upxeik2Db7XjPCooCjvBINiAw6Nq+i7oMo9nbG8ecDs=";
+  };
+
+  doCheck = false;
+  vendorSha256 = "sha256-fOlfVI8v7KqNSRhAPlZBSHKfZRlCbCgjnMV/6bsqDhg=";
+
+  meta = with lib; {
+    description = "This application is a NetFlow/IPFIX/sFlow collector in Go.";
+    homepage = "https://github.com/cloudflare/goflow";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ heph2 ];
+    platforms = platforms.linux;
```

Why only Linux? Go can be compile to other OSs too. Or is the program only written for Linux? I couldn't find anything about that in their readme. I suggest:
```suggestion
    platforms = platforms.all;
```

---

In which case platforms could be dropped entirely.

---

https://nixos.org/manual/nixpkgs/stable/#var-meta-platforms says that there won't be prebuilt binaries then though.